src/utils/components/DescriptionItem/DescriptionItem.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import MutedTextSpan from '@kubevirt-utils/components/MutedTextSpan/MutedTextSpan';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListTermHelpText,
+  DescriptionListTermHelpTextButton,
+  Flex,
+  FlexItem,
+  Popover,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+
+import './DescriptionItem.scss';
+
+type DescriptionItemProps = {
+  descriptionData: any;
+  descriptionHeader?: string;
+  bodyContent?: React.ReactNode;
+  moreInfoURL?: string;
+  isPopover?: boolean;
+  breadcrumb?: string;
+  isEdit?: boolean;
+  onEditClick?: () => void;
+  isDisabled?: boolean;
+  showEditOnTitle?: boolean;
+  'data-test-id'?: string;
+};
+
+const DescriptionItem: React.FC<DescriptionItemProps> = ({
+  descriptionData,
+  descriptionHeader,
+  bodyContent,
+  moreInfoURL,
+  isPopover,
+  breadcrumb,
+  isEdit,
+  onEditClick,
+  isDisabled,
+  showEditOnTitle,
+  'data-test-id': testId,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const NotAvailable = <MutedTextSpan text={t('Not available')} />;
```

Comment:
i think we already have this kind of generic component... please check. @avivtur

Reply:
i copied @avivtur 's component and made it generic. we should use this one instead of his